Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8
```

Comment:
Maybe using just python image could be heavy, so why don't we use alpine instead?

Reply:
i recommend `python` image with `3.8-alpine` tag.